src/fromager/__main__.py

```diff
@@ -195,13 +209,19 @@ def main(
             logger.info(f"variant: {variant}")
             logger.info(f"patches dir: {patches_dir}")
             logger.info(f"maximum concurrent jobs: {jobs}")
-            logger.info(f"constraints file: {constraints_file}")
+            logger.info(
+                f"constraints files: {os.pathsep.join(str(p) for p in constraints_file)}"
+            )
+            logger.info(f"constraints URLs: {' '.join(constraints_url)}")
             logger.info(f"network isolation: {network_isolation}")
             overrides.log_overrides()
 
     if network_isolation and not SUPPORTS_NETWORK_ISOLATION:
         ctx.fail(f"network isolation is not available: {NETWORK_ISOLATION_ERROR}")
 
+    constraints: list[str] = list(constraints_url)
+    constraints.extend(os.fspath(p) for p in constraints_file)
```

Comment:
This implies that constraints at URLs take precedence over local files. I don't think that ordering is wrong, but I can definitely imagine wanting a local file to have a value that replaces a value in some global file on a web server.